Algeria.json

```diff
@@ -2318,6 +2335,7 @@
             },
             {
                "matricule": "19- Setif",
+               "phoneCode": "34",

```

Comment:
According to the [same source](https://fr.wikipedia.org/wiki/Liste_des_codes_t%C3%A9l%C3%A9phoniques_des_wilayas_d%27Alg%C3%A9rie), Setif's code should be `36`.